packages/fiori/src/SideNavigationItemBase.ts

```diff
@@ -40,12 +40,14 @@ class SideNavigationItemBase extends UI5Element implements ITabbable {
 
 	/**
 	 * Defines the tooltip of the component.
+	 *
+	 * A tooltip attribute should be provided, in order to represent meaning/function, when the component is collapsed(icon only is visualized).
 	 * @default ""
-	 * @private
-	 * @since 1.0.0-rc.16
+	 * @public
+	 * @since 1.24
```

Comment:
main branch is 2.0 - since should be updated
and actually this is not a breaking change but enhancement, feature - the commit msg should be updated